workflows/reference_files/2.4/reference_files.smk

```diff
@@ -60,6 +60,23 @@ rule create_gatk_dict:
         """)
 
 
+rule generate_transcriptome:
```

Comment:
Can you incorporate different genome builds?

Reply:
Sure. I only did this because I thought we want to count everything against the same reference

Reply:
This is definitely only for the sake of non-GAMBL users/projects.